### PR TITLE
RAAS fixes

### PR DESCRIPTION
**Pull request by theotherjimmy**

### CODE REVIEW

**Comment by theotherjimmy** [comment]
Why is circle not running?

**Comment by theotherjimmy** [comment]
Oh, there's travis!

File: mbed_greentea/mbed_greentea_cli.py [hunk 1]
```diff
@@ -344,6 +344,12 @@ def main():
                     action="store_true",
                     help='Prints console outputs for failed tests')
 
+    parser.add_option('', '--retry-test',
+                    dest='retry_test',
+                    default=5, # ** SET IT 1 , ITS USED only for testing**
```
Comment (cmonr) [inline mbed_greentea/mbed_greentea_cli.py]: Does this comment need to be updated?

Comment (theotherjimmy) [reply mbed_greentea/mbed_greentea_cli.py]: Yep!

**Comment by theotherjimmy** [comment]
@studavekar Would you like to take this over?

**Comment by studavekar** [comment]
Have created separate PR for above changes. Closing it.

**Pull request closed without merging**